### DISCUSSION TITLE
How to Fix a basic import error?

### DISCUSSION THREAD

**Discussion by EindbaasExpress** [com-lihaoyi/mill#1650] [Q&A] [answered]
Hey There, 

I'm fairly new to Scala and build tools. 
Via Haoyi's book I learned about Mill and the examples on build pipelines are very interesting.

For a side project I wanted to import a Java Package created by Google. But I keep getting an import error: 
`object google is not a member of package com`

I have the project working in an SBT-project, but I can't stomach that it doesn't work in Mill.
I have created a small example project [here](https://github.com/EindbaasExpress/gtm_audit_mill) to show the set up.

What am I doing wrong?

P.S. I am using VS Code with Metals and the project is based on 'example-3' [this one](https://github.com/com-lihaoyi/mill/tree/main/example/example-3)

Best,
Arne

**Comment by lefou** [comment, marked as answer]
I'm just quoting your `build.sc` for context:

```scala
// build.sc
import mill._, scalalib._

object GtmAudit extends ScalaModule{
  def scalaVersion = "2.13.7"
  object test extends Tests{
    def ivyDeps = Agg(ivy"com.lihaoyi::utest:0.7.10",
                      ivy"com.google.api-client:google-api-client:1.31.5")
    def testFrameworks = Seq("utest.runner.Framework")
  }
}
```
 
You only defined the google library as dependency of the `GtmAudit.test` module, whereas you also want it to use in `GtmAudit` module. 
You need to move the dependency from the test module to the core module.

```scala
// build.sc
import mill._, scalalib._

object GtmAudit extends ScalaModule{
  def scalaVersion = "2.13.7"
  def ivyDeps = Agg(ivy"com.google.api-client:google-api-client:1.31.5")
  object test extends Tests{
    def ivyDeps = Agg(ivy"com.lihaoyi::utest:0.7.10")
    def testFrameworks = Seq("utest.runner.Framework")
  }
}
```

**Reply by EindbaasExpress** [reply]
Wow, that's fast.
And that indeed makes total sense.
Thank you very much for pointing that out!
